sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/SqlDelightExtension.kt

```diff
@@ -1,11 +1,39 @@
 package com.squareup.sqldelight.gradle
 
-import org.gradle.api.file.FileCollection
-import java.io.File
-
-open class SqlDelightExtension(
-  var packageName: String? = null,
```

Comment:
Can we keep these and synthesize the new model when used? Right now this is a breaking API change which is pretty dramatic as a consumer. We can mark them deprecated and emit a warning in the Gradle build when they're used.